user-guide/Advanced_Functionality/Failover/Configuring_Failover/Preparing_the_two_DataMiner_Agents.md

```diff
@@ -15,7 +15,7 @@ Before you start the actual configuration, make sure you have the following:
   > [!NOTE]
   > If you use IP addresses instead of a hostname:
   >
-  > - Make sure [NPCap](https://nmap.org/npcap/) or WinPCap (deprecated) is installed on both DMAs (see [Before you run the installer](xref:Installing_DM_using_the_DM_installer#before-you-run-the-installer)).
+  > - Make sure [Npcap](https://nmap.org/npcap/) or WinPcap (deprecated) is installed on both DMAs (see [Before you run the installer](xref:Installing_DM_using_the_DM_installer#before-you-run-the-installer)). From DataMiner 10.4.10/10.5.0 onwards<!--RN 40257 + RN 40267-->, you can check if these Pcap libraries are installed by reviewing the [Cube logging](xref:Cube_logging). Each time the *Failover Config* window is opened, a *PcapInfoRequestMessage* is sent to the DataMiner Agent. The response, tagged with the keyword "PcapInfo", can be found in Cube logging alongside an `Info` entry. This message will only be sent if the [*Configure Failover* user permission](xref:DataMiner_user_permissions#modules--system-configuration--agents--configure-failover) is enabled for the user.
```

Comment:
Perhaps also change "pcap libraries" to "packet capture libraries"

Reply:
```suggestion
  > - Make sure [Npcap](https://nmap.org/npcap/) or WinPcap (deprecated) is installed on both DMAs (see [Before you run the installer](xref:Installing_DM_using_the_DM_installer#before-you-run-the-installer)). From DataMiner 10.4.10/10.5.0 onwards<!--RN 40257 + RN 40267-->, you can check if these packet capture libraries are installed by reviewing the [Cube logging](xref:Cube_logging). Each time the *Failover Config* window is opened, a *PcapInfoRequestMessage* is sent to the DataMiner Agent. The response, tagged with the keyword "PcapInfo", can be found in Cube logging alongside an `Info` entry. This message will only be sent if the [*Configure Failover* user permission](xref:DataMiner_user_permissions#modules--system-configuration--agents--configure-failover) is enabled for the user.
```